File: CS/StandaloneReportManagerServiceExample/ViewModels/MainViewModel.cs
using DevExpress.Mvvm;
using System;
using System.Collections.Generic;
using System.Collections.ObjectModel;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace StandaloneReportManagerServiceExample.ViewModels {
    public class MainViewModel : ViewModelBase {
        public List<ItemViewModel> Items {
            get { return GetProperty(() => Items); }
            set { SetProperty(() => Items, value); }
        }

        public MainViewModel() {
            Items = new List<ItemViewModel>() {
                new ItemViewModel() { ID = 1, Name = "First" },
                new ItemViewModel() { ID = 2, Name = "Second" },
                new ItemViewModel() { ID = 3, Name = "Third" },
                new ItemViewModel() { ID = 4, Name = "Fourth" },
            };
        }
    }

    public class ItemViewModel : ViewModelBase {
        public string Name {
            get { return GetProperty(() => Name); }
            set { SetProperty(() => Name, value); }
        }
        public int ID {
            get { return GetProperty(() => ID); }
            set { SetProperty(() => ID, value); }
        }
    }
}

File: VB/StandaloneReportManagerServiceExample/ViewModels/MainViewModel.vb
Imports DevExpress.Mvvm
Imports System.Collections.Generic

Namespace StandaloneReportManagerServiceExample.ViewModels

    Public Class MainViewModel
        Inherits ViewModelBase

        Public Property Items As List(Of ItemViewModel)
            Get
                Return GetProperty(Function() Me.Items)
            End Get

            Set(ByVal value As List(Of ItemViewModel))
                SetProperty(Function() Items, value)
            End Set
        End Property

        Public Sub New()
            Items = New List(Of ItemViewModel)() From {New ItemViewModel() With {.ID = 1, .Name = "First"}, New ItemViewModel() With {.ID = 2, .Name = "Second"}, New ItemViewModel() With {.ID = 3, .Name = "Third"}, New ItemViewModel() With {.ID = 4, .Name = "Fourth"}}
        End Sub
    End Class

    Public Class ItemViewModel
        Inherits ViewModelBase

        Public Property Name As String
            Get
                Return GetProperty(Function() Me.Name)
            End Get

            Set(ByVal value As String)
                SetProperty(Function() Name, value)
            End Set
        End Property

        Public Property ID As Integer
            Get
                Return GetProperty(Function() Me.ID)
            End Get

            Set(ByVal value As Integer)
                SetProperty(Function() ID, value)
            End Set
        End Property
    End Class
End Namespace
